lib/galaxy/webapps/galaxy/api/jobs.py

```diff
@@ -185,6 +186,29 @@ def outputs( self, trans, id, **kwd ):
         job = self.__get_job( trans, id )
         return self.__dictify_associations( trans, job.output_datasets, job.output_library_datasets )
 
+    @expose_api_anonymous
+    def build_for_rerun( self, trans, id, **kwd ):
+        """
+        * GET /api/jobs/{job_id}/build_for_rerun
```

Comment:
I just noticed that we both used `{job_id}` here while the function parameter and the doc below use `id`, should we standardize on the latter? All public methods in this file have the same problem.